packages/plugin-essentials/sources/commands/set/version.ts

```diff
@@ -189,7 +189,7 @@ export async function setVersion(configuration: Configuration, bundleVersion: st
 
     const manifest = (await Manifest.tryFind(projectCwd)) || new Manifest();
 
-    if (bundleVersion && miscUtils.isTaggedYarnVersion(bundleVersion))
+    if (bundleVersion && semver.valid(bundleVersion) && miscUtils.isTaggedYarnVersion(bundleVersion))
```

Comment:
The right fix is rather to fix `isTaggedYarnVersion` - its regex currently matches everything that doesn't contain a `-` 😅